### PR TITLE
Added a build image for NodeJS and FFMPEG.

### PR DESCRIPTION
**Pull request by FenrirWillow**
This PR adds a new image to our collection.

It is a base NodeJS image that contains FFMPEG inside it as well. We use this as a base image for our video rendering servers.

### CODE REVIEW

File: ffmpeg-node/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# FFMPEG Node Docker image
+### `dubit/ffmpeg-node`
+This is a utility image that contains the latest Node.js runtime plus the ffmpeg command line utility. This image can be used for containers that transcode, transmux or do other video file operations in conjuction with Node.js
+
```
Comment (pmead) [inline ffmpeg-node/README.md:4]: Maybe append a heads up here about just how much goes into this container and a warning about size.

File: ffmpeg-node/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+# * build process requires a CMake version higher than 3.5.
+RUN wget https://cmake.org/files/v3.11/cmake-3.11.1-Linux-x86_64.tar.gz && tar -xzf cmake-3.11.1-Linux-x86_64.tar.gz
+ENV PATH="${PATH}:/cmake-3.11.1-Linux-x86_64/bin/"
+
+# --- FFMPEG Build and install ---
+RUN mkdir -p ~/ffmpeg_sources ~/bin
+
+# --- Install nasm assembler ---
+# * This is an assembler used by some of FFMPEG's peer dependencies.
+RUN cd ~/ffmpeg_sources && \
```
Comment (pmead) [inline ffmpeg-node/Dockerfile:29]: There are a few common folders and variables in here that could be moved into vars to make updates to this less error prone

File: ffmpeg-node/README.md (hunk 1 above)
Comment (LukeAvery92) [reply ffmpeg-node/README.md:4]: Is it always the latest node or the latest at the time?

**Comment by FenrirWillow** [comment]
We ultimately went into a different direction internally, as well as this ending up being a little too complicated and large. I will revisit it at a later date and see if I can improve it

**Pull request closed without merging**